central/vulnerabilityrequest/manager/requestmgr/manager_impl_cache_test.go

```diff
@@ -1,142 +1,136 @@
 package requestmgr
```

Comment:
Added all tests to a suite. Since the test works with mocks, init once and reuse is okay.